apps/communications/dialer/test/marionette/lib/views/activities/accessors.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/* global module */
+
+var SELECTORS = Object.freeze({
+  addNewContact: 'button[data-l10n-id="createNewContact"]',
+  addToExistingContactMenuItem: 'button[data-l10n-id="addToExistingContact"]'
+  //cancelButton
+  //Check if main is needed here
+
+});
+
+function ActivitiesAccessor(client) {
+  this.client = client;
+}
+
+ActivitiesAccessor.prototype = {
+  get addNewContact() {
```

Comment:
Nit: As we return HtmlElement to the view, I'd say which type of element is returned, so the person who reads the views knows that he gets something he can tap on. For example: `addNewContactMenuItem`.